src/nni_manager/training_service/test/localTrainingService.test.ts

```diff
@@ -72,6 +74,38 @@ describe('Unit Test for LocalTrainingService', () => {
         chai.expect(jobDetail.status).to.be.equals('USER_CANCELED');
     }).timeout(20000);
 
+    it('Get trial log', async () => {
+        // set meta data
+        const trialConfig: string = `{\"command\":\"python3 mockedTrial.py\", \"codeDir\":\"${localCodeDir}\",\"gpuNum\":0}`
+        await localTrainingService.setClusterMetadata(TrialConfigMetadataKey.TRIAL_CONFIG, trialConfig);
+
+        // submit job
+        const form: TrialJobApplicationForm = {
+            sequenceId: 0,
+            hyperParameters: {
+                value: 'mock hyperparameters',
+                index: 0
+            }
+        };
+
+        const jobDetail: TrialJobDetail = await localTrainingService.submitTrialJob(form);
+
+        // get trial log
+        const rootDir: string = getExperimentRootDir()
+        fs.mkdirSync(path.join(rootDir, 'trials'))
+        fs.mkdirSync(jobDetail.workingDirectory)
+        fs.writeFileSync(path.join(jobDetail.workingDirectory, 'trial.log'), 'trial log')
+        fs.writeFileSync(path.join(jobDetail.workingDirectory, 'stderr'), 'trial stderr')
+        chai.expect(await localTrainingService.getTrialLog(jobDetail.id, 'TRIAL_LOG')).to.be.equals('trial log');
+        chai.expect(await localTrainingService.getTrialLog(jobDetail.id, 'TRIAL_ERROR')).to.be.equals('trial stderr');
+        fs.unlinkSync(path.join(jobDetail.workingDirectory, 'trial.log'))
+        fs.unlinkSync(path.join(jobDetail.workingDirectory, 'stderr'))
+        fs.rmdirSync(jobDetail.workingDirectory)
+        fs.rmdirSync(path.join(rootDir, 'trials'))
+
+        await localTrainingService.cancelTrialJob(jobDetail.id);
+    }).timeout(20000);
```

Comment:
this is testing training service, do we need to also test nnimanager or restserver?

Reply:
This UT for `getTrialLog` looks OK for me, we only want to test getTrialLog here, we do not care about other parts.

Reply:
ok